temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
+        if file_type == "txt":
+            output_file.write("")
+        elif file_type == "json":
+            output_file.write(json.dumps({}))
+        else:
+            log.error("ERROR: Invalid file type")
```

Comment:
Instead of logging an error when an unsupported file type is provided, consider raising an exception. This will make it clear that an error has occurred and will allow the caller to handle it appropriately. [medium]